bundles/org.eclipse.emfcloud.modelserver.client/src/org/eclipse/emfcloud/modelserver/client/ModelServerClient.java

```diff
@@ -59,17 +59,17 @@
 
 public class ModelServerClient implements ModelServerClientApi<EObject>, ModelServerPaths, AutoCloseable {
 
-   private static final Set<String> DEFAULT_SUPPORTED_FORMATS = ImmutableSet.of(ModelServerPathParameters.FORMAT_JSON,
+   protected static final Set<String> DEFAULT_SUPPORTED_FORMATS = ImmutableSet.of(ModelServerPathParameters.FORMAT_JSON,
       ModelServerPathParameters.FORMAT_XMI);
-   private static final String PATCH = "PATCH";
-   private static final String POST = "POST";
+   protected static final String PATCH = "PATCH";
+   protected static final String POST = "POST";
 
-   private static Logger LOG = Logger.getLogger(ModelServerClient.class.getSimpleName());
+   protected static Logger LOG = Logger.getLogger(ModelServerClient.class.getSimpleName());
```

Comment:
Shouldn't subclasses declare their own logger instance?

Reply:
Yes, you're right, that makes more sense! I changed it back to private